tests/__sf_section_rust.test.fish

```diff
@@ -47,7 +47,7 @@ end
 test "Doesn't print the section when Cargo.toml and *.rs aren't present"
 	(
 		rm -rf /tmp/tmp-spacefish/Cargo.toml
-	) = (__sf_section_golang)
+	) = (__sf_section_rust)
```

Comment:
Noticed a little slip-up in the rust tests.
A little drive-by fix. 🚗

Reply:
Interesting how this didn't come up with an error or something.
Nice spot! :eyes: